docs/sources/administration/team-management/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+aliases:
+  - 
+description: This document introduces Grafana Teams and concepts.
```

Comment:
I think the description could be revised to better suit that it is presented to the user as part of search engine results.

Reply:
@jdbaldry I am not sure how you'd like to re-word the description, it's in line with most other descriptions in this repo.

Reply:
It's probably OK, but for the future, keep in mind the guidance in https://grafana.com/docs/writers-toolkit/write/front-matter/#description-required.